PLBenchmarks/targets.py

```diff
@@ -0,0 +1,373 @@
+"""
+targets.py
+Functions and classes for handling the target data.
+"""
+
+import os
+import yaml
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import networkx as nx
+
+from . import __path__, ligands, edges, utils
+
+
+data_path = os.path.abspath(
+    os.path.join(os.path.join(__path__[0], "sample_data"))
+)
+file = open(os.path.join(data_path, "targets.yml"))
+target_dict = yaml.full_load(file)
+print(target_dict)
+file.close()
+
+
+def set_data_dir(
```

Comment:
If this function is only used to serve `/sample_data/` to the tests and not for the module without the sample data, consider moving it to the test file

Reply:
Ok, and then set `target_dict` and `data_path` to `None`? I set it to sample_data, that you have all the functionalities right from the beginning. Then users won't get frustrated.